github-archive/README.md

```diff
@@ -27,3 +27,18 @@ data provided with Crate. Requires ```npm``` to work.
 npm install
 npm start
 ´´´
+
+# Prepare AWS Credentials for importing data
+
+Provide credentials:
+
+```sh
+  export AWS_ACCESS_KEY_ID='...'
+  export AWS_SECRET_ACCESS_KEY='...'
+```
+
+Import data from s3 to the Crate cluster:
+
+```sh
+python import.py --start 2011/8 --end 2011/10 --host localhost:4200
```

Comment:
this is not the buildout way to invoke scripts.
please add a console script entry point in `setup.py`
see: https://github.com/crate/crash/blob/master/setup.py#L71